packages/sanitize/src/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Strip HTML tags.
+ *
+ * @param {string} text Text to have the HTML tags striped out of.
```

Comment:
Fixed in https://github.com/WordPress/packages/pull/12/commits/6c93f801d6fd0138491e434775f13382b1bfad68